packages/next/src/generators/page/page.ts

```diff
@@ -27,13 +27,12 @@ export async function pageGeneratorInternal(host: Tree, schema: Schema) {
   const options = await normalizeOptions(host, schema);
   const componentTask = await reactComponentGenerator(host, {
     ...options,
-    project: schema.project,
-    pascalCaseFiles: false,
+    nameAndDirectoryFormat: 'as-provided', // already determined the directory so use as is
+    project: undefined,
```

Comment:
this is marked as `undefined` because we are using `projectName` instead and `project` will be removed in Nx 18 right?

Reply:
yeah exactly, just ignore it

Reply:
Yeah, I updated to make them optional. I can probably remove this line.